lms/djangoapps/email_marketing/migrations/0006_auto_20170711_0615.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('email_marketing', '0005_emailmarketingconfiguration_user_registration_cookie_timeout_delay'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='emailmarketingconfiguration',
+            name='user_registration_cookie_timeout_delay',
+            field=models.FloatField(default=3.0, help_text='The number of seconds to delay/timeout wait to get cookie values from sailthru.'),
```

Comment:
This will only change the default value for new entries, it will no update old ones that already exist in the database. Is that acceptable?

```
mysql> select distinct user_registration_cookie_timeout_delay from email_marketing_emailmarketingconfiguration;
+----------------------------------------+
| user_registration_cookie_timeout_delay |
+----------------------------------------+
|                                    1.5 |
|                                      3 |
+----------------------------------------+
2 rows in set (0.00 sec)
```